src/plugins/vis_type_timeline/server/index.ts

```diff
@@ -45,6 +45,7 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
     renameFromRoot('timeline_vis.enabled', 'vis_type_timeline.enabled'),
     renameFromRoot('timeline.enabled', 'vis_type_timeline.enabled'),
     renameFromRoot('timeline.graphiteUrls', 'vis_type_timeline.graphiteUrls'),
+    renameFromRoot('vis_type_timeline.graphiteUrls', 'vis_type_timeline.graphiteAllowedUrls'),
```

Comment:
just curious: so this actually does the work of addressing any opensearch_dashboards.yml changes? as long as its defined here then even if it we renamed values as long as we have the original it will read it?

Reply:
I tried `timeline.graphiteUrls`, `vis_type_timeline.graphiteUrls` and `vis_type_timeline.graphiteAllowedUrls` and they all work. Only difference is that if use `timeline.graphiteUrls` or `vis_type_timeline.graphiteUrls`, `yarn start` will throw deprecation warnings. I think this could be an easy way to address opensearch_dashboards.yml changes for customer migrate.